samples/cellular/lte_ble_gateway/README.rst

```diff
@@ -1,198 +0,0 @@
-.. _lte_sensor_gateway:
```

Comment:
This label is referenced in the old release notes, [cloud doc](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/external_comp/nrf_cloud.html#applications_and_samples), and Known issues page. These ref links need to be removed from cloud docs and replaced with plain text in the old release notes and the Known issue page. This fixes the doc build.